src/bin/utils/options.ts

```diff
@@ -4,7 +4,7 @@
  * @module
  */
 import commander from 'commander';
-import * as binParsers from './parsers';
+import * as parsers from '../utils/parsers';
```

Comment:
Keep this as `binParsers`. We do this pattern in other domains like `clientErrors`... etc.

Reply:
Changed back.